v0/destinations/intercom/deleteUsers.js

```diff
@@ -0,0 +1,50 @@
+const { httpSend } = require("../../../adapters/network");
+const { CustomError } = require("../../util");
+
+const responseHandler = async (userAttributes, config) => {
```

Comment:
I believe this should be named something more appropriately, maybe `userDeletionHandler`.